tests/jballerina-unit-test/src/test/resources/test-src/expressions/builtinoperations/freeze-and-isfrozen.bal

```diff
@@ -456,15 +455,6 @@ function testValidSelfReferencingValueFreeze() returns [string, boolean] {
     return [FREEZE_SUCCESSFUL, !m.isReadOnly() && res.isReadOnly()];
 }
 
-function testStructureWithErrorValueFreeze() returns boolean {
```

Comment:
Why is this removed?

Reply:
This test was checking whether an error is readonly. As per spec, isReadOnly() accepts only anydata. Hence, removed because the test is not applicable

Reply:
I think we can you `res is readonly` here. 
Created a spec issue to clarify why that function is restricted to `anydata`.  https://github.com/ballerina-platform/ballerina-spec/issues/769

Reply:
Once https://github.com/ballerina-platform/ballerina-lang/issues/23691 get fixed, cloneReadOnly() would anyway return   `CloneableType & readonly` which would make `res is readonly` always true

Reply:
We are removing that `always true` error with https://github.com/ballerina-platform/ballerina-lang/issues/28285

Reply:
What I meant was even though the compiler validation for always true will be removed; I believe the testing scenario is not correct as we assert something that's always true. WDYT?

Reply:
`is` test is a runtime check, so we verify the readonly-ness of the value (whether the readonly bit is set or not in the cloned value). 
Not sure how it's always true, as we are testing the behavior of `cloneReadOnly`.

Reply:
I believe that due to the current restriction on error casting we cannot do a is check with `Cloneable` of `map<anydata|error>`. May we have to disable this test case only temporarily

Reply:
My confusion was with the below sample as the `if check` will always be true. Clarified it from @rdhananjaya and added the test back in e3df68af6b6da7ae7b9a4e539c815fd41e26cf56.  For the moment, commented out the `is readonly` check. The test should be updated once #23691 is fixed.
```ballerina
import ballerina/io;

public function main() {
    var er = getReadOnlyError();
    if(er is readonly) {
        io:println("er is readonly");
    }
}

function getReadOnlyError() returns error & readonly {
    return error("Custom error");
}
```